pkgs/misc/vscode-extensions/default.nix

```diff
@@ -582,6 +582,20 @@ let
         };
       };
 
+      redhat.java = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "java";
+          publisher = "redhat";
+          version = "0.76.0";
+          sha256 = "0xb9brki4s00piv4kqgz6idm16nk6x1j6502jljz7y9pif38z32y";
+        };
+        buildInputs = [ jdk ];
+        meta = {
+          license = lib.licenses.epl20;
+          broken = lib.versionAtLeast "11" jdk.version;
```

Comment:
Just to make sure: Does this need jdk of at least 11.0 or does it support 11 and below?

Reply:
<https://github.com/redhat-developer/vscode-java/blob/master/README.md>

> Download and install a recent Java Development Kit (latest Java 11 is the current minimum requirement).

Reply:
#117291

Reply:
Thank you